src/test/fixtures/plugin-kotlin-js/expected/build/reports/licensee/artifacts.json

```diff
@@ -10,5 +10,53 @@
                 "url": "https://www.apache.org/licenses/LICENSE-2.0"
             }
         ]
+    },
+    {
+        "groupId": "org.jetbrains.kotlin",
+        "artifactId": "kotlin-dom-api-compat",
+        "version": "1.9.0",
+        "name": "Kotlin Dom Api Compat",
+        "spdxLicenses": [
+            {
+                "identifier": "Apache-2.0",
+                "name": "Apache License 2.0",
+                "url": "https://www.apache.org/licenses/LICENSE-2.0"
+            }
+        ],
+        "scm": {
+            "url": "https://github.com/JetBrains/kotlin"
+        }
+    },
+    {
+        "groupId": "org.jetbrains.kotlin",
+        "artifactId": "kotlin-stdlib-common",
```

Comment:
The problematic dependency (without Gradle metadata) is added/resolved using Kotlin js plugin.